apps/system/utils/utils_stackmonitor.c

```diff
@@ -132,15 +132,9 @@ static void stkmon_print_title(void)
 static void stkmon_print_inactive_list(void)
 {
 	int inactive_idx;
-	struct stkmon_save_s terminated_infos[STKMON_MAX_LOGS];
-
-	/* Initialize the terminated_infos array. */
-	for (inactive_idx = 0; inactive_idx < (STKMON_MAX_LOGS); inactive_idx++) {	
-		terminated_infos[inactive_idx].timestamp = 0;
-	}
 
 	/* Copy the terminated task/pthread information from kernel. */
-	prctl((int)PR_GET_STKLOG, (struct stkmon_save_s *)terminated_infos);
+	prctl((int)PR_GET_STKLOG, terminated_infos);
```

Comment:
prctl checks some of configs internally, and return error if each of config is not enabled.
Is it safe with out return value?